code/modules/mob/animations.dm

```diff
@@ -327,9 +333,17 @@ note dizziness decrements automatically in the mob's Life() proc.
 
 /atom/movable/proc/do_putdown_animation(atom/target, mob/user)
 	spawn()
+		if (!istype(src) && !QDELETED(src))
+			return
+		if (!istype(target) && !QDELETED(target))
+			return
+		if (!istype(user) && !QDELETED(user))
+			return
```

Comment:
Shouldn't 
`if (!istype(user) && !QDELETED(user))`
 be 
`if (!istype(user) || QDELETED(user))`
?

Reply:
Yes